### ISSUE TITLE
Add option to use annotations

### ISSUE THREAD

**Issue by littlechicks** [Jaydee94/kubeseal-webgui#144]
Thanks you for the tools.
Great job !!

Could we improve it by adding annotation handler ?

In fact Sealed Secret encrypt data but also annotations. So adding annotation on SealedSecret object will not add it when unseal it.
We have to  add first annotation on the Secret ressource and only then we can seal it in order to take annotation in count.

Can we improve the GUI for letting us add some annotations like :

```
sealedsecrets.bitnami.com/managed
sealedsecrets.bitnami.com/namespace-wide
sealedsecrets.bitnami.com/cluster-wide
```

**Comment by Jaydee94** [comment]
We`ll have a look on that.

**Comment by Jaydee94** [comment]
closed with #153
